eq-author/src/App/introduction/Design/IntroductionEditor/CollapsiblesEditor/CollapsibleEditor/index.js

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import { flowRight } from "lodash";
+import styled from "styled-components";
+import gql from "graphql-tag";
+import PropTypes from "prop-types";
+import { propType } from "graphql-anywhere";
+
+import DeleteButton from "components/buttons/DeleteButton";
+import MoveButton, { IconUp, IconDown } from "components/buttons/MoveButton";
+import RichTextEditor from "components/RichTextEditor";
+import { colors } from "constants/theme";
+
+import withChangeUpdate from "enhancers/withChangeUpdate";
+import withPropRenamed from "enhancers/withPropRenamed";
+import withEntityEditor from "components/withEntityEditor";
+
+import withUpdateCollapsible from "./withUpdateCollapsible";
+import withDeleteCollapsible from "./withDeleteCollapsible";
+
+const Detail = styled.div`
+  border: 1px solid ${colors.bordersLight};
+  padding: 0 1em;
```

Comment:
I think this would be a better layout:

Part 1

```suggestion
  padding: 2em 1em 0;
```